rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/RealWorldGroovyTest.java

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.groovy.tree;
+
+import org.junit.jupiter.api.Test;
+import org.junitpioneer.jupiter.ExpectedToFail;
+import org.openrewrite.Issue;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.groovy.Assertions.groovy;
+
+/**
+ * A test with groovy examples picked from OSS repositories.
+ */
+class RealWorldGroovyTest implements RewriteTest {
+
+    @Test
+    @ExpectedToFail("Pattern operator is not yet supported") // https://groovy-lang.org/operators.html#_pattern_operator
+    @Issue("https://github.com/spring-projects/spring-boot/blob/main/settings.gradle")
```

Comment:
Can we update these links to make them stable in light of any future changes?
```suggestion
    @Issue("https://github.com/spring-projects/spring-boot/blob/v3.4.1/settings.gradle")
```